Xamarin.Forms.Platform.Tizen/FormsApplication.cs

```diff
@@ -49,7 +50,24 @@ protected override void OnPreCreate()
 		{
 			base.OnPreCreate();
 			Application.ClearCurrent();
-			MainWindow = PreloadedWindow.GetInstance() ?? new Window("FormsWindow");
+			var type = typeof(Window);
+			// Use reflection to avoid breaking compatibility. ElmSharp.Window.CreateWindow() is has been added since API6.
+			var methodInfo = type.GetMethod("CreateWindow", BindingFlags.NonPublic | BindingFlags.Static);
```

Comment:
Yeah, reflection is generally slow. But if you already know the type and just extract the `MethodInfo`, it's not that too slow. When tested in a Tizen products on the market, it was less than `1ms`. And, as mentioned in the comments, `CreateWidnow()` is an API added since Tizen5.5, so even if we get the platform version, we will get a compile error because the type is not present in versions prior to 5.5. This is why we used reflection to ensure compatibility.